string-hash/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for string-hash 1.1.1
+// Project: https://github.com/darkskyapp/string-hash
+// Definitions by: Ethan Rubio <https://github.com/ethanrubio/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace StringHash {
```

Comment:
Actually, looking at the [code](https://github.com/darkskyapp/string-hash/blob/master/index.js) this is not made available as a global variable. So you can leave off the `export as namespace` too.